src/main/java/teammates/storage/entity/Instructor.java

```diff
@@ -52,7 +51,8 @@
     @Unindex
     private String displayedName;
 
-    private Text instructorPrivilegesAsText;
+    @Unindex
+    private String instructorPrivileges;
```

Comment:
This is backward-incompatible and will require data migration. Let's not change the field name.